diff --git a/dist/lib/store/index.js b/dist/lib/store/index.js
index fa20718fb080636258e11f494a0c34a3e70a93a3..52c2a01821e4e5838cf99db50e45fce84f579dfa 100644
--- a/dist/lib/store/index.js
+++ b/dist/lib/store/index.js
@@ -1,4 +1,4 @@
-import { panBy as panBySystem, updateNodeInternals as updateNodeInternalsSystem, initialConnection, errorMessages, updateAbsolutePositions, snapPosition, calculateNodePosition, getHandlePosition, Position } from '@xyflow/system';
+import { panBy as panBySystem, updateNodeInternals as updateNodeInternalsSystem, initialConnection, errorMessages, updateAbsolutePositions, snapPosition, calculateNodePosition, getHandlePosition, Position, withResolvers } from '@xyflow/system';
 import { addEdge as addEdgeUtil } from '../utils/edges.js';
 import { initialEdgeTypes, initialNodeTypes, getInitialStore } from './initial-store.svelte.js';
 import {} from './types.js';
@@ -77,7 +77,7 @@ export function createStore(signals) {
     function fitView(options) {
         // We either create a new Promise or reuse the existing one
         // Even if fitView is called multiple times in a row, we only end up with a single Promise
-        const fitViewResolver = store.fitViewResolver ?? Promise.withResolvers();
+        const fitViewResolver = store.fitViewResolver ?? withResolvers();
         // We schedule a fitView by setting fitViewQueued and triggering a setNodes
         store.fitViewQueued = true;
         store.fitViewOptions = options;
diff --git a/dist/lib/store/initial-store.svelte.d.ts b/dist/lib/store/initial-store.svelte.d.ts
index 61a1524bcb13b1d80d7f2a8d4e5f0a4a70aa6bcd..a988a3c01742268145dd822dcd522fd5bf3cbf0f 100644
--- a/dist/lib/store/initial-store.svelte.d.ts
+++ b/dist/lib/store/initial-store.svelte.d.ts
@@ -1,6 +1,7 @@
 import { SelectionMode, ConnectionMode, type SelectionRect, type SnapGrid, type MarkerProps, type PanZoomInstance, type CoordinateExtent, type NodeOrigin, type OnError, type Viewport, type OnConnect, type OnConnectStart, type OnConnectEnd, type NodeLookup, type ConnectionState, type EdgeLookup, type ConnectionLookup, type ParentLookup, type ColorModeClass, type Handle, type OnReconnect, type OnReconnectStart, type OnReconnectEnd, type AriaLabelConfig, type ZIndexMode } from '@xyflow/system';
 import type { NodeTypes, EdgeTypes, FitViewOptions, OnDelete, OnBeforeConnect, OnBeforeDelete, IsValidConnection, Edge, Node, EdgeLayouted, InternalNode, OnBeforeReconnect, OnSelectionChange, OnSelectionDrag } from '../types/index.js';
 import type { StoreSignals } from './types.js';
+import type { withResolvers } from '@xyflow/system';
 import { MediaQuery } from 'svelte/reactivity';
 export declare const initialNodeTypes: {
     input: import("svelte").Component<import("../types/index.js").NodeProps, {}, "">;
@@ -64,7 +65,7 @@ export declare function getInitialStore<NodeType extends Node = Node, EdgeType e
     connectionDragThreshold: number;
     fitViewQueued: boolean;
     fitViewOptions: FitViewOptions<NodeType> | undefined;
-    fitViewResolver: PromiseWithResolvers<boolean> | null;
+    fitViewResolver: ReturnType<typeof withResolvers<boolean>> | null;
     snapGrid: SnapGrid | null;
     dragging: boolean;
     selectionRect: SelectionRect | null;
